UnityProject/Assets/Scripts/ScriptableObjects/Audio/FloorSounds.cs

```diff
@@ -1,14 +1,18 @@
-﻿using System.Collections;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using AddressableReferences;
 using UnityEngine;
 
-[CreateAssetMenu(fileName = "FloorSounds", menuName = "ScriptableObjects/FloorSounds")]
-public class FloorSounds : ScriptableObject
+namespace ScriptableObjects.Audio
 {
-	public List<AddressableAudioSource> Barefoot = new List<AddressableAudioSource>();
-	public List<AddressableAudioSource> Claw = new List<AddressableAudioSource>();
-	public List<AddressableAudioSource> Shoes = new List<AddressableAudioSource>();
+	[CreateAssetMenu(fileName = "FloorSounds", menuName = "ScriptableObjects/FloorSounds")]
+	public class FloorSounds : ScriptableObject
+	{
+		public List<AddressableAudioSource> Barefoot = new List<AddressableAudioSource>();
+		public List<AddressableAudioSource> Claw = new List<AddressableAudioSource>();
+		public List<AddressableAudioSource> Shoes = new List<AddressableAudioSource>();
+		public List<AddressableAudioSource> Clown = new List<AddressableAudioSource>();
```

Comment:
the sounds clown shoes would make would be defined on the clown shoes not the tile

Reply:
It is an easier way to override it. Say you want to make clown play water splashes when stepping on water tile, you just assign the water splashes on its sound category and forget about doing exceptions.

Reply:
if I’m not mistaken even ss13 defined the sounds in the tiles themselves

Reply:
well, there's a bool to make the tile Override any custom sounds like Clown  to play its own

Reply:
but then you have to individually set overrides in an SO and assign it to the step changer which will store a variable in Mind.cs . It was wacky when I first did step changer, it wasn't supposed to stay like that.

Reply:
It seems simple though, if you want clown shoes to make clown sounds add the class StepChanger  to clown shoes and it works, if the tile however has a custom sound and tile wants to override then it can,